[FEAT] 온보딩 페이지 구현

HyeonChae1104 opened the pull request:
## 주요 변경사항 ✔️
온보딩 화면을 구현하였습니다. 

- 온보딩 페이지에 들어갈 이미지 추가
- 카카오 로그인 디자인 가이드에 맞춘 버튼 추가
- 기능 4가지 설명을 위한 네비게이션을 구현하였습니다.

온보딩 -> 로그인 -> 메인화면으로 가는 로직은 이후 로그인 페이지 작업 후 추가하도록 하겠습니다!

## 리뷰어에게... 🙏
온보딩 페이지를 이렇게도 바꿔보고, 저렇게도 바꿔봤는데 디자인이 너무 어렵네요..하하
보시고 코드 수정이 필요해보이는 부분이나 디자인이 바뀌었으면 좋겠는 곳들을 위주로 봐주시면 좋을 것 같아요...! 

## 관련 이슈
closes #31

Da-Yeon-Kim committed:
remove: CRA로 생긴 파일 정리

- public 폴더 내 파일 (logo, favicon, manifest.json)
- 기본 CSS 파일 (index.css, App.css)
- 성능 측정 도구 (reportWebVitals.js)
- 테스트 파일 (App.test.js)

 #18

Da-Yeon-Kim committed:
[REMOVE] CRA로 생긴 파일 정리

geongyu09 committed:
docs: pr 템플릿 내용 변경 (#30)

pr 템플릿을 약간 수정하였습니다

HyeonChae1104 committed:
feat(onboarding): 온보딩 화면에 들어갈 이미지와 아이콘 추가

- 온보딩 화면에서 서비스의 기능 설명을 위한 사진을 추가하였습니다.
- 카카오 로그인 버튼을 위한 아이콘을 추가하였습니다.

#31

HyeonChae1104 committed:
feat(onboarding): 온보딩 화면을 구현

- 온보딩 과정에 사용할 이미지를 추가
- 제목, 부제목, 카카오 로그인 버튼을 추가
- 기능 설명을 위해 화살표를 통한 네비게이션 기능을 구현했습니다.

#31

HyeonChae1104 committed:
feat(router): 온보딩 화면을 위한 라우트 설정 추가

- 온보딩 화면으로 이동할 수 있는 라우트를 추가했습니다.

#31

geongyu09 reviewed (approved):
와우 완벽한데요??
디자인이 어렵다니요.. 이렇게나 이쁜데

geongyu09 commented:
추가적으로 브랜치 Weekly5로 변경해주세요!

Da-Yeon-Kim reviewed (approved):
와, 디자인이 진짜 예쁜데요? 넘겨 보는 아이디어가 멋져요~ 사용하신 이미지는 추후에 다 완성되면 바꾸시는 건가요? 잊지 않게 저도 기억해두겠습니다,, 고생하셨어요! ☺

Da-Yeon-Kim commented on src/pages/Onboarding/index.tsx:
```diff
@@ -0,0 +1,190 @@
+  font-size: 2rem;
+  color: #333;
+  margin-bottom: 1rem;
+  font-family: 'Noto Sans KR', sans-serif;
```
근데 혹시 저희 폰트 이걸로 통일하나요? 일단 이쪽에 맞추어서 스타일링 할까요!

HyeonChae1104 replied:
일단 깔끔한 느낌으로 해본건데 통일할 폰트를 이야기해서 정하고 변경하는것도 좋을 것 같아요~!ㅎㅎ

Da-Yeon-Kim replied:
저는 Figma에서 Pretendard로 한 것 같아서 그 폰트로 우선 작업하고 있긴 했습니다~

HyeonChae1104 replied:
아! 맞네요!! 제가 Pretendard로 수정했습니다!

Da-Yeon-Kim reviewed (approved):
아, 살펴보다가 추가로 생각난 건데, 화면 크기가 작을 때는 이미지가 위로 올라가게끔 반응형으로 디자인하면 보기 더 좋을 것 같아요~

HyeonChae1104 committed:
chore(onboarding): 폰트 변경

폰트를 Pretendard로 변경하였습니다.

#31

HyeonChae1104 committed:
feat(onboarding): 작은 화면에서 적용되는 반응형 디자인 추가

- 768px 이하의 화면에서 이미지가 위로 올라가도록 @media 쿼리 추가
- 화면이 작을 때 레이아웃이 수직으로 정렬되도록 수정하였습니다.

#31
